templates/data/whitesource-periodics-data.yaml

```diff
@@ -100,6 +100,54 @@ templates:
                     - "jobConfig_periodic"
                   local:
                     - "jobConfig_default"
+              # Kyma components
+              - jobConfig:
+                  repository: "kyma-operator"
+                  language: "golang-mod"
+                  project: "kyma-project"
+                  subprojects: "false"
+                  base_ref: "main"
+                inheritedConfigs:
+                  global:
+                    - "jobConfig_default"
+                    - "image_wssagent"
+                    - "extra_refs_test-infra"
+                    - "jobConfig_periodic"
+                  local:
+                    - "jobConfig_default"
+                    - "product_kyma"
+              - jobConfig:
+                  repository: "kyma-watcher"
```

Comment:
I think we should split the watcher scan into skr and kcp as they are 2 go.mod and two internal projects in monorepo style. Is that possible?

Reply:
resolved in https://github.com/kyma-project/test-infra/pull/5796